### PR TITLE
Update to new osl-firewall resources

### PR DESCRIPTION
**Pull request by detjensrobert**

### CODE REVIEW

**Comment by detjensrobert** [comment]
[osl-ceph](https://trello.com/c/pOvojbNe/2105-osl-ceph)

File: resources/osl_cephfs.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@
   end
 
   mons = ceph_chef_mon_addresses.sort.join(',') + ':' + new_resource.subdir
+  mons += '/' if Chef::VERSION.to_i == 16
```
Comment (ramereth) [inline resources/osl_cephfs.rb:24]: Workaround for https://github.com/chef/chef/issues/10764

**Review by ramereth — APPROVED** [review]

**Comment by detjensrobert** [comment]
!bump major

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v4.0.0.  Have a nice day!

**Pull request merged**